politeiawww/user/cockroachdb/models.go

```diff
@@ -48,6 +48,21 @@ func (User) TableName() string {
 	return tableUsers
 }
 
+// Session represents a politeiawww user session.
+type Session struct {
+	ID     uuid.UUID `gorm:"primary_key"`            // UUID
+	UserID uuid.UUID `gorm:"not null"`               // User UUID (foreign key)
+	MaxAge int64     `gorm:"not null;default:86400"` // default: one day
```

Comment:
It's a default -- when the `Session` record is inserted we can still specify any `MaxAge` value we want.

Reply:
What's the point in having a hard coded default though?  It opens the possibility of someone in the future relying on the default value, then if we decide to change the value of the sessionMaxAge variable we've got problems because its unlikely that they'll remember to also update the cockroachdb model.

Reply:
Alright. I have removed the default value as requested.